### ISSUE TITLE
Script does not delete "down" hosts

### ISSUE THREAD

**Issue by Romuss** [lopes/netbox-scanner#15]
Hi!
Through this point:
`The script will iterate through each network to find and delete any hosts registered in NetBox that did not respond to scan, and have the tag defined in the configuration file.`
i can't understand, how script understands that host is down and script should delete this host.
I tried some scenarios, but nothing works.

**Comment by Romuss** [comment]
```
[GENERAL]
tag       = test
unknown   = unknown host
log       = .
nmap_args = -p 22,53,80,443


[NETBOX]
address = https://netbox-test
token = fadbdf2179d438f086
tls_verify = False

[TACACS]
user     = 
password = 
command  = show run | inc hostname
regex    = hostname ([A-Z|a-z|0-9|\-|_]+)
regroup  = 1

[SCAN]
networks = 10.25.100.0/24
```

**Comment by xPakrikx** [comment]
yop. problem is this commit. https://github.com/lopes/netbox-scanner/commit/0b03893204a2d9487e3789cb157a19f09ba2707c

**Comment by lopes** [comment]
Yeah, it was pointed out in #6 . It'll be fixed on the next version, see #18 , thanks.
